test/QuadForm/Quad/ZGenus.jl

```diff
@@ -264,6 +264,17 @@
         q2, _ = normal_form(q2)
         q2 = Hecke.gram_matrix_quadratic(q2)
         @test q1 == q2
+<<<<<<< HEAD
```

Comment:
There are some merge artifacts here.

Reply:
Oh, sorry I didn't realise it earlier. Will make changes and push them soon. Thanks